src/pspm_cfg/pspm_cfg_run_dcm.m

```diff
@@ -93,5 +97,7 @@
 if eventnameflag
   options.eventnames = eventnames;
 end
-[sts, out] = pspm_dcm(model, options);
+pspm_dcm(model, options);
```

Comment:
I am not sure if it is valid to write it in this way... If `model` is not assigned in the output of `pspm_dcm`, it will remain unchanged. Please see the experiment below
<img width="285" alt="image" src="https://github.com/user-attachments/assets/aa4aee8b-42cd-423f-8d2f-9b5b99e9f449">

Reply:
Yes, this is valid because `pspm_dcm` will not change `model.modelfile`.